pipelines/respondent-home-ui.yml

```diff
@@ -142,7 +142,7 @@ jobs:
         HOST: 0.0.0.0
         LOG_LEVEL: INFO
         ACCOUNT_SERVICE_URL: http://rhpreprod.rmdev.onsdigital.uk
-        EQ_URL: https://preprod-surveys.eq.ons.digital/session?token=
+        EQ_URL: https://eq.onsdigital.uk/session?token=
```

Comment:
Shouldn't `/session?token=` be part of the application not the environment variable? as it doesn't change per environment?

Reply:
@jonnyshaw89 we kept it as an environment variable in case runner changed the pathing. But you could be right that we can hardcode it for the social tests.

Reply:
Its not changed in last 3 years so doubt it will. also, if it did change there it would probably be part of a bigger interface change so presumably code changes would be required anyway?

Reply:
@jonnyshaw89 cool. We'll get the path set at a code level.

Reply:
Let's do this as a separate PR to get this in

Reply:
@benjefferies @jonnyshaw89 being done as part of https://github.com/ONSdigital/respondent-home-ui/pull/35 and #56

Reply:
Ok I've updated it here @jcox-dev